templates/nodeovc/schema.capnp

```diff
@@ -0,0 +1,76 @@
+@0xd37ff48ad935931f;
+struct Schema {
+	# Description for the VM
+	description @0 :Text;
+
+	# Name of the VM
+	name @1 :Text;
+	
+	# OpenvCloud
+	openvcloud @2 :Text;
+
+	# OS Image
+	osImage @3 :Text = "Ubuntu 16.04";
+
+	# Memory available for the vm in GB
+	bootDiskSize @4 :Int64 = 10;
+
+	# Type of VM: defines the number of CPU and memory available for the vm
```

Comment:
In the code I see we only use sizeId, right?
So vcpus and memory if to foreseen the future or ?

Reply:
In `js9` currently `memory` and `vcpus` are present as parameters of the function `machine_create`, however, the values seem to not be chosen freely and must be derived from sizeId. I don't know why they are in the parameters
``` python
    def machine_create(
            self,
            name,
            sshkeyname=None,
            memsize=2,
            vcpus=None,
            disksize=10,
            datadisks=[],
            image="Ubuntu 16.04 x64",
            sizeId=None,
            stackId=None,
            sshkeypath=None,
            description=None,
    ):
```